instrumentation/dubbo/src/test/java/brave/dubbo/ITTracingFilter_Consumer.java

```diff
@@ -240,6 +247,52 @@
 
     Span span =
       reporter.takeRemoteSpanWithError(Span.Kind.CLIENT, ".*Not found exported service.*");
-    assertThat(span.tags().get("dubbo.error_code")).isEqualTo("NETWORK_EXCEPTION");
+    assertThat(span.tags())
+      .containsEntry("rpc.error_code", "NETWORK_EXCEPTION");
+  }
+
+  /** Shows if you aren't using RpcTracing, the old "dubbo.error_code" works */
+  @Test public void addsErrorTag_onUnimplemented_legacy() {
+    ((TracingFilter) ExtensionLoader.getExtensionLoader(Filter.class)
+      .getExtension("tracing")).isInit = false;
+
+    ((TracingFilter) ExtensionLoader.getExtensionLoader(Filter.class)
+      .getExtension("tracing"))
+      .setTracing(tracing);
+
+    assertThatThrownBy(() -> wrongClient.get().sayHello("jorge"))
+      .isInstanceOf(RpcException.class);
+
+    Span span =
+      reporter.takeRemoteSpanWithError(Span.Kind.CLIENT, ".*Not found exported service.*");
+    assertThat(span.tags())
+      .containsEntry("dubbo.error_code", "1");
```

Comment:
FYI, we hadn't released the change to the legacy dubbo error code format. I put it back when RpcTracing is not in use, and deprecated the Tracing setter (cc @anuraaga)